src/content/app-platform/getting-started/index.md

```diff
@@ -0,0 +1,260 @@
+latest version according to semantic versioning.
+
+We are also filtering using the catalog label. You can change this to
+`giantswarm-playground` to show our more experimental apps which we encourage
+you to try but do not offer as a managed service.
+
+```sh
+kubectl get appcatalogentry -l latest=true,application.giantswarm.io/catalog=giantswarm
+
+NAME                                             CATALOG      APP NAME                       APP VERSION   VERSION        AGE
+...
+giantswarm-nginx-ingress-controller-app-1.16.1   giantswarm   nginx-ingress-controller-app   v0.45.0       1.16.1         25h
+...
+```
+
+## Creating an App CR
+
+We can use the [kubectl gs template app]({{< relref "/ui-api/kubectl-gs/login" >}})
+command to generate the App CR using the latest version from the previous command.
+
+```sh
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster=${NAMESPACE} \
+  --name=nginx-ingress-controller-app \
+  --namespace=kube-system \
+  --version=1.16.1 > nginx-ingress-controller-app.yaml
+
+kubectl apply -f nginx-ingress-controller-app.yaml
+```
+
+Lets first see the output of the template command which shows only the
+required fields.
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: nginx-ingress-controller-app
+  namespace: s2wf9
+spec:
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: nginx-ingress-controller-app
+  namespace: kube-system
+  version: 1.16.1
+```
+
+## Defaulting and App Status
+
+Now lets check the app using the `kubectl gs get app` command which filters
+the output to show only the most important fields.
+
+```sh
+kubectl -n ${NAMESPACE} gs get app nginx-ingress-controller-app -o yaml
+```
+
+The labels, cluster config and kubeconfig have been all defaulted to the correct
+values for your cluster. You can read more about defaulting and validation for
+App CRs [here]({{< relref "/app-platform/defaulting-validation" >}}).
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 3.2.1
+    app.kubernetes.io/name: nginx-ingress-controller-app
+  name: nginx-ingress-controller-app
+  namespace: s2wf9
+spec:
+  catalog: giantswarm
+  config:
+    configMap:
+      name: ingress-controller-values
+      namespace: s2wf9
+  kubeConfig:
+    context:
+      name: s2wf9
+    inCluster: false
+    secret:
+      name: s2wf9-kubeconfig
+      namespace: s2wf9
+  name: nginx-ingress-controller-app
+  namespace: kube-system
+  version: 1.16.1
+status:
+  appVersion: v0.45.0
+  release:
+    lastDeployed: "2021-04-21T16:28:08Z"
+    status: deployed
+  version: 1.16.1
+```
+
+In the App CR status you can see that the app is deployed. The `appVersion`
+shows that this version of the app is deploying `v0.45.0` of the upstream
+[Nginx Ingress Controller](https://github.com/kubernetes/ingress-nginx) project.
```

Comment:
@cokiengchiara I added this so we can close https://github.com/giantswarm/giantswarm/issues/11992

Reply:
perfect thank you!

Reply:
thinking if we should state what `version` is but maybe that's obvious.

Reply:
I think version is OK. The problem is appVersion because lots of things are called app 😂